docker-compose.yml

```diff
@@ -25,11 +25,11 @@ x-sentry-defaults: &sentry_defaults
   volumes:
     - 'sentry-data:/var/lib/sentry/files'
 x-snuba-defaults: &snuba_defaults
+  restart: unless-stopped
```

Comment:
It should be `restart: always`. Otherwise system shutdown will cause it to not be brought back after reboot.